mcm/tools/locator.py

```diff
@@ -1,51 +1,134 @@
 import os
+import sys
+import logging
 
 
 class locator:
+    """
+    Groups all the required configuration variables for the web application
+    """
+
     def __init__(self):
-        pass
+        self.logger = self.__create_logger()
 
-    def isDev(self):
-        return not (self.isInt() or self.isProd())
+    def __create_logger(self):
```

Comment:
No need for a separate function, do this in `__init__()`

Reply:
The function `isDev`? If so, I would avoid removing it at this moment. It is used in different places of the McM web application source code. In future iterations, we could scan these kinds of pitfalls and refactor them.

Reply:
No, this was the same as the other comment about `__create_logger`

Reply:
Okay, thanks for the clarification. I would like to keep this as it is and the explanation is the same for the others.